miniapp/USERGUIDE.md

```diff
@@ -163,7 +163,26 @@ val miniAppMessageBridge = object: MiniAppMessageBridge() {
         
         callback.invoke(true, null) // or callback.invoke(false, "error message")
     }
+
+// initialize UserInfoHandler after ensuring that MiniApp has been created.
```

Comment:
What happens if they don't create the Mini App first? It seems inconvenient because they need to create `MiniAppMessageBridge` before calling `MiniApp.create`, but then they're supposed to set the `UserInfoHandler` after calling `MiniApp.create`.

Reply:
It will show error as `MiniAppCustomPermissionCache` hasn't been initialized internally. I had initialize `UserInfoHandler` inside `MiniAppDisplayViewModel.obtainMiniAppDisplay(~)` which is not user friendly.